blog/2015-03-19-JBDS-8.1.0.CR1-for-luna.adoc

```diff
@@ -0,0 +1,52 @@
+= JBoss Developer Studio 8.1.0.CR1 and JBoss Tools 4.2.3.CR1 
```

Comment:
Should we also fix the Beta1 blog then? https://github.com/nickboldt/jbosstools-website/blob/release423CR1/_config/products.yml#L70